lib/github-release.rb

```diff
@@ -119,7 +120,7 @@ def create_release(tag)
 
 		api.create_release(repo_name, tag, :name => name, :body => body)
 
-		puts " done!"
+		puts "done!"
```

Comment:
Yes, but that's how I always handle pseudo-ellipsis and spacing.  I also always use Oxford commas ;-)